### DISCUSSION TITLE
Running R Markdown documents with RStudio and publish them via GitHub, using GitHub Pages

### DISCUSSION THREAD

**Discussion by TanishqKaushik** [community/community#67765] [New to GitHub]
Hi, 

I am trying to figure out how to display R Markdown files on GitHub online. I have gone through this entire tutorial - "https://resources.github.com/github-and-rstudio/" a few times however, every time I try to create a new branch and make local changes with git using this code - "knit: (function(input_file, encoding) {
    out_dir <- 'docs';
    rmarkdown::render(input_file,
      encoding=encoding,
      output_file=file.path(dirname(input_file), out_dir, 'index.html'))})" 
I get an error stating that the directory does not exist. I have tried using getwd() and setwd() to confirm if the directory is right however it still says that the directory does not exist. 

I have tried manually adding the docs folder in the directory to populate it with the html output but git on RStudio does not even recognize it as a change that needs to be committed.

**Comment by queenofcorgis** [comment]
Hi there 👋🏼 - I'm going to look into this for you, hang tight!

**Reply by TanishqKaushik** [reply]
Thank you!

**Reply by TanishqKaushik** [reply]
Were you able to figure it out queenofcorgis?

**Reply by queenofcorgis** [reply]
Hi there @TanishqKaushik thanks for your patience I did do some digging on this error, given that this is R Studio I'd recommend heading to their community to ask this as they may be better suited to help: https://community.rstudio.com/.

**Reply by TanishqKaushik** [reply]
Thanks for the suggestion @queenofcorgis!

**Reply by queenofcorgis** [reply]
You're welcome - I hope they can help!
